pkg/distributors/test/MerkleOrchard.test.ts

```diff
@@ -301,6 +307,46 @@ describe('MerkleOrchard', () => {
         merkleOrchard.connect(admin).createDistribution(token1.address, root2, claimableBalance.mul(2), 1)
       ).to.be.revertedWith(errorMsg);
     });
+
+    context('when claiming the same distribution ID twice within the same request', () => {
+      sharedBeforeEach('create a distribution with 2 users and twice the claimable balance', async () => {
+        distributionId = 2;
+        elements = [
+          encodeElement(claimer1.address, claimableBalance),
+          encodeElement(claimer2.address, claimableBalance),
+        ];
+        merkleTree = new MerkleTree(elements);
+        const root = merkleTree.getHexRoot();
+
+        await merkleOrchard
+          .connect(distributor)
+          .createDistribution(token1.address, root, claimableBalance.mul(2), distributionId);
+        merkleProof = merkleTree.getHexProof(elements[0]);
+
+        claims = [
```

Comment:
While not an issue here, this is not a good pattern. You share the `merkleTree`, `claims`, etc variables accross multiple scopes, but here set them to a different value. If you don't have a similar `beforeEach` block that sets them in other scopes, they'll be affected by these changes.

I find it better to restrict the scope of the variables to where they're set/used to avoid these things (even if in this case that'd mean every block having their own version of `claims` etc)